wrappers/react-native/example/.detoxrc.json

```diff
@@ -1,13 +1,25 @@
 {
-  "testRunner": "jest",
-  "runnerConfig": "jest.config.js",
-  "skipLegacyWorkersInjection": true,
+  "testRunner": {
+    "$0": "jest",
+    "args": {
+      "config": "jest.config.js",
+      "_": [
+        "e2e"
+      ]
+    },
+    "forwardEnv": true
+  },
+  "jest": {
+    "setupTimeout": 120000,
+    "reportSpecs": false,
+    "reportWorkerAssign": false
+  },
   "apps": {
     "ios.release": {
       "name": "PairingCryptoRnExample",
       "type": "ios.app",
       "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/PairingCryptoRnExample.app",
-      "build": "xcodebuild -workspace ios/PairingCryptoRnExample.xcworkspace -scheme PairingCryptoRnExample -configuration Release -sdk iphonesimulator -derivedDataPath ios/build ENTRY_FILE=index.tsx EXCLUDED_ARCHS=arm64 DETOX=1"
```

Comment:
Entry file has changed 😄 to index.js

https://github.com/mattrglobal/pairing_crypto/pull/231/files#r1875587057